### ISSUE TITLE
Arbitrary "HTML" report types

### ISSUE THREAD

**Issue by metasim** [deepsense-ai/seahorse#17]
It would be very powerful if there were a generic "HTML" report type where an `DOperation` could generate information in some other form than tables.

**Comment by jaroslaw-osmanski** [comment]
Would Notebook generating HTML be a solution for you?

Example code: 

from IPython.display import HTML
HTML("<h1>Hello</h1>")

You can send an email from Seahorse notebook containing HTML as attachment. Reference: https://seahorse.deepsense.ai/operations/python_notebook.html

**Comment by metasim** [comment]
@jaroslaw-osmanski No, this is at the custom operation level, where `Report` or `ReportContent` can deliver something other than tables, like a text summary, images, external links, etc. when you click on the Report icon of a node. This  could be done with HTML, but Markdown would also suffice. We are building custom analytics nodes where we still want pre-canned summary reports, but of a different form.

**Comment by metasim** [comment]
Here's another use case: our DataFrames often have very complex schemas (nested stucts, etc). I'd like to have a report type which renders the schema in hierarchical tree format, with the nodes being links to external documentation on the context or provenance of the data, rather than just the column centric form currently offered.
